[Bug]: 1.20.1 The error says that the json file is empty

### Latest Version

- [X] I am running the latest version of the mod.

### What happened?

Crash Game

> [Render thread/ERROR] [Seamless/]: Couldn't parse data file environmental:lily_pad_center from environmental:seamless_rules/lily_pad_center.json
> com.google.gson.JsonParseException: java.io.EOFException: End of input at line 1 column 1 path $
> 
> Caused by: java.io.EOFException: End of input at line 1 column 1 path $
> 	at com.google.gson.stream.JsonReader.nextNonWhitespace(JsonReader.java:1455) ~[gson-2.10.jar%2372!/:?] {}
> 	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:558) ~[gson-2.10.jar%2372!/:?] {}
> 	at com.google.gson.stream.JsonReader.peek(JsonReader.java:433) ~[gson-2.10.jar%2372!/:?] {}
> 	at com.google.gson.internal.bind.TypeAdapters$28.read(TypeAdapters.java:769) ~[gson-2.10.jar%2372!/:?] {}
> 	at com.google.gson.internal.bind.TypeAdapters$28.read(TypeAdapters.java:725) ~[gson-2.10.jar%2372!/:?] {}
> 	at com.google.gson.internal.bind.TypeAdapters$34$1.read(TypeAdapters.java:1007) ~[gson-2.10.jar%2372!/:?] {}
> 	at net.minecraft.util.GsonHelper.m_13780_(GsonHelper.java:524) ~[client-1.20.1-20230612.114412-srg.jar%231002!/:?] {re:classloading,pl:connector_pre_launch:A,re:mixin,pl:connector_pre_launch:A}
> 	... 64 more

### How to reproduce the bug?

In version 2.2.0, there may not have been such an error.

### What version of Minecraft are you running?

1.20.1

### What mod loader are you using?

Forge

### latest.log

_No response_

--- a/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/cattail.json
+++ b/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/cattail.json
@@ -1,0 +1,5 @@
+{
+  "blocks": ["environmental:cattail"],
+  "directions": ["down"],
+  "connecting_blocks": ["environmental:cattail_stalk"]
+}

--- a/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/cattail_stalk.json
+++ b/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/cattail_stalk.json
@@ -1,0 +1,5 @@
+{
+  "blocks": ["environmental:cattail_stalk"],
+  "directions": ["up", "down"],
+  "connecting_blocks": ["/same", "environmental:cattail"]
+}

--- a/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/double_high_lower.json
+++ b/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/double_high_lower.json
@@ -1,5 +1,5 @@
 {
-  "blocks": ["environmental:tall_cattail", "environmental:tall_dead_bush", "environmental:giant_tall_grass"],
+  "blocks": ["environmental:giant_tall_grass"],
   "blockstates": {
     "half": ["lower"]
   },

--- a/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/double_high_upper.json
+++ b/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/double_high_upper.json
@@ -1,5 +1,5 @@
 {
-  "blocks": ["environmental:tall_cattail", "environmental:tall_dead_bush", "environmental:giant_tall_grass"],
+  "blocks": ["environmental:giant_tall_grass"],
   "blockstates": {
     "half": ["upper"]
   },

--- a/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/lily_pad_center.json
+++ b/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/lily_pad_center.json
@@ -1,0 +1,8 @@
+{
+  "blocks": ["environmental:giant_lily_pad"],
+  "blockstates": {
+    "position": ["center"]
+  },
+  "directions": ["north", "east", "south", "west"],
+  "connecting_blocks": ["/same"]
+}

--- a/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/lily_pad_east.json
+++ b/forge/src/main/resources/resourcepacks/default_seamless/assets/environmental/seamless_rules/lily_pad_east.json
@@ -1,7 +1,7 @@
 {
   "blocks": ["environmental:giant_lily_pad"],
   "blockstates": {
-    "position": ["northeast"]
+    "position": ["east"]
   },
   "directions": ["north", "south", "west"],
   "connecting_blocks": ["/same"]

